cmd/convert.go

```diff
@@ -103,6 +105,7 @@ func init() {
 
 	// OpenShift only
 	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", true, "Generate an OpenShift deploymentconfig object")
+	convertCmd.Flags().BoolVar(&ConvertInsecureRepo, "insecure-repository", false, "Use Insecure Repository")
```

Comment:
No need for capitalization! Correct grammar is `Use an insecure repository`.

It might be better to add a note that this is OpenShift only for now. So perhaps `Use an insecure OpenShift repository`, would be better.